src/Microsoft.Extensions.DataIngestion.Abstractions/Chunk.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.Extensions.DataIngestion
+{
+    [DebuggerDisplay("{Content}")]
+    public sealed class Chunk
```

Comment:
nit: in the near future we might rename it to `DocumentChunk` as other types use the `Document` prefix.

```suggestion
    public sealed class DocumentChunk
```